### PR TITLE
Added cards component

### PR DESCRIPTION
**Pull request by Shah-Arsalan**
Added the cards component

### CODE REVIEW

File: components/card.css [hunk 1]
```diff
@@ -0,0 +1,256 @@
+@import url(/main.css);
+@import url('https://fonts.googleapis.com/css2?family=Raleway:ital,wght@1,300&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Raleway:ital,wght@1,300&family=Source+Code+Pro:ital,wght@1,200&display=swap');
+
+*{
```
Comment (Aman-Jain1996) [inline components/card.css]: shift it in main.css

File: components/card.css [hunk 2]
```diff
@@ -0,0 +1,256 @@
...
+    flex-direction: column;
+    justify-content: space-between;
+    /* margin: 5rem 0; */
+    background-color: white;
+   height: min-content;
+
+}
+
+.image{
+    width: 100%;
```
Comment (Aman-Jain1996) [inline components/card.css:15]: height : 100%

File: components/card.css [hunk 3]
```diff
@@ -0,0 +1,256 @@
...
+
+.footer-button{
+    display: flex;
+    justify-content: space-around;
+    align-content: center;
+    padding-top: 1rem;
+
+}
+
+.card-button{
```
Comment (Aman-Jain1996) [inline components/card.css:27]: cursor : pointer

**Review by Aman-Jain1996 — COMMENTED** [review]
Do required changes

File: ImageFolder/cards/card.html [hunk 4]
```diff
@@ -0,0 +1,167 @@
...
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>components</title>
+    <!-- <link rel="stylesheet" href="/card1.css"> -->
+    <link rel="stylesheet" href="/components/card.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.10.0/css/all.min.css" integrity="sha512-PgQMlq+nqFLV4ylk1gwUOgm6CtIIXkKwaIHp/PAIWHzig/lKZSEGKEysh0TCVbHJXCLN7WetD8TFecIky75ZfQ==" crossorigin="anonymous" referrerpolicy="no-referrer" />
+<body>
+
+    <div class="megaContainer">
+
```
Comment (thegeorgenikhil) [inline ImageFolder/cards/card.html:14]: class names should not be camelCase 
use `mega-container`

**Review by thegeorgenikhil — COMMENTED** [review]
Hey the spacing looks very disorganized 
Try doing a prettier formatting before pushing the code
The code will look much better!

File: components/card.css [hunk 5]
```diff
@@ -0,0 +1,256 @@
...
+    width: 100%;
+   background-color: var(--primary-color);
+    padding: 1rem;
+    border: transparent;
+}
+
+.content{
+    padding-left: 1rem;
+
+    /* font-family: 'Raleway', sans-serif; */
```
Comment (anuj-kumary) [inline components/card.css]: Don't add unnecessary comments and also format the code (Prettier , Beautify)

File: components/card.css [hunk 6]
```diff
@@ -0,0 +1,256 @@
...
+
+}
+
+.small{
+        font-size: small;
+    }
+    
+.card-content{
+        /* font-family: 'Raleway', sans-serif; */
+        font-family: 'Open Sans', sans-serif;
```
Comment (vinc3nati) [inline components/card.css:49]: Specify a font size, style in the root and then inherit it wherever required.

File: components/card.css (hunk 5 above)
Comment (Shah-Arsalan) [reply components/card.css]: fixed , can you check

File: ImageFolder/cards/card.html (hunk 4 above)
Comment (Shah-Arsalan) [reply ImageFolder/cards/card.html:14]: removed that mega container and fixed spacing . Can you approve please?

File: components/card.css (hunk 2 above)
Comment (Shah-Arsalan) [reply components/card.css:15]: done , can you check now?

**Review by thegeorgenikhil — APPROVED** [review]
Approving this now.
Use prettier at all times👍🏻

For CSS  it is not recommended to put more than one line spacing b/w subsequent classes, at most should be two
Thank you

**Review by vinc3nati — APPROVED** [review]
Looks good

File: components/card.css (hunk 2 above)
Comment (Aman-Jain1996) [reply components/card.css:15]: Looks good to be merged

File: components/card.css (hunk 1 above)
Comment (Aman-Jain1996) [reply components/card.css]: Good to be merged

**Pull request merged**